Tensile/Source/lib/include/Tensile/ContractionProblem.hpp

```diff
@@ -725,14 +736,15 @@ namespace Tensile
         std::string m_sumNames;
         std::string m_operationIdentifier;
 
-        bool           m_transA;
-        bool           m_transB;
-        bool           m_stridedBatched          = true;
-        bool           m_highPrecisionAccumulate = false;
-        bool           m_deterministicMode       = false;
-        bool           m_eligibleForPK           = true;
-        ArithmeticUnit m_arithmeticUnit          = ArithmeticUnit::Any;
-        KernelLanguage m_kernelLanguage          = KernelLanguage::Any;
+        bool            m_transA;
+        bool            m_transB;
+        bool            m_stridedBatched          = true;
+        bool            m_highPrecisionAccumulate = false;
+        bool            m_deterministicMode       = false;
+        bool            m_eligibleForPK           = true;
+        ArithmeticUnit  m_arithmeticUnit          = ArithmeticUnit::Any;
+        BenchmarkMetric m_benchmarkMetric         = BenchmarkMetric::Overall;
+        KernelLanguage  m_kernelLanguage          = KernelLanguage::Any;
```

Comment:
Same as above - ContractionProblem is used in many places outside of benchmarking, might be more intuitive to call it 'performance metric'

Reply:
I don't have a strong preference here. I'll make the change to PerformanceMetric unless someone else has strong feelings on what it should be called.